README.md

```diff
@@ -1,5 +1,8 @@
-## A project is a simple take to write netowrking scripts.
+## Python Netowrking scripts.
 
 #### Netwkring protocols this project will build servers and clinets for:
   - HTTP, TCP, UDP ,WebSockets, WSS, DNS, and MQTT.
   - info.py:  a simple information displayer for domains, IPv4 addresses and ports, it will the main inforamtion gathering script. 
+  - server.py: will be a http server that serves content and deal with conneciton.
+  - client.py: will be the client to connect to the server and send requests for files and render them . 
+  - dns-lookup.py: dedicated script for DNS protocol.
```

Comment:
what code style this project will follow? include it here.

Reply:
added the PEP8 style guide that this project will follow for each python program.

Reply:
Good